pandas/core/frame.py

```diff
@@ -8040,3 +8040,13 @@ def _from_nested_dict(data):
 
 def _put_str(s, space):
     return u'{s}'.format(s=s)[:space].ljust(space)
+
+
```

Comment:
don't put this here, pandas/core/indexing.py is better or pandas/core/index/api.py

Reply:
`_get_info_slice` is only used once in `frame.py` and nowhere else. I was thinking we might as well avoid the import.

Reply:
ok, can you just inline it then where it is used (make it a local function) instead.